src/client/createTRPCNuxtClient.ts

```diff
@@ -20,6 +20,16 @@ interface ResolverDef {
   errorShape: any;
 }
 
+// Extracted from https://github.com/trpc/trpc/blob/5597551257ad8d83dbca7272cc6659756896bbda/packages/client/src/internals/TRPCUntypedClient.ts#L32
+interface TRPCSubscriptionObserver<TValue, TError> {
```

Comment:
it loses types when importing directly from `@trpc/client/dist/internals/TRPCUntypedClient`